### PR TITLE
9029 - Fixed glossary to open from URL

### PR DESCRIPTION
**Pull request by anjenkin**
https://federal-spending-transparency.atlassian.net/browse/DA-9029

### CODE REVIEW

File: src/components/headers/mobile-menu.jsx [hunk 1]
```diff
@@ -21,6 +21,16 @@ class MobileMenu extends React.Component {
 			glossaryClickCount: 0,
 		};
 	}
+	componentDidMount() {
+		if (window.location.search.indexOf('glossary') > -1) {
+			const urlParams = new URLSearchParams(window.location.search);
+			const myParam = urlParams.get('glossary');
```
Comment (brianpetway) [inline src/components/headers/mobile-menu.jsx:27]: It looks like this term isn't used. Can it be deleted?

**Comment by brianpetway** [comment]
See one inline comment.
The page reloads when clicking on a glossary term in the copy; that isn't in the tkt but is that supposed to happen?
The url is not retaining the search term after the glossary opens.
If you paste in a url with a search term, the page opens with the glossary open , but the url changes to no longer include the search term after the page loads

**Comment by anjenkin** [comment]
@brianpetway Yeah I added code to remove the search term from the URL because that's what USAS is doing.  If I leave the term in the URL, when opening the glossary the term will remain selected which isn't expected I would think.  

Yes, the page reload thing is in a separate ticket.

Good catch, I'l remove that unneeded code.

**Comment by anjenkin** [comment]
@brianpetway Fixed

**Pull request merged**